stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -265,15 +265,16 @@ public class MessageListViewModel(
      */
     init {
         observeTypingUsers()
+        observeMessages()
         observeChannel()
     }
 
     /**
-     * Starts observing the current channel. We observe the 'messagesState', 'user' and 'endOfOlderMessages'
-     * states, as well as build the `newMessageState` using [getNewMessageState] and combine it
-     * into a [MessagesState] that holds all the information required for the screen.
+     * Starts observing messages of the current channel. We observe the 'messagesState', 'user' and
+     * 'endOfOlderMessages' states, as well as build the `newMessageState` using [getNewMessageState]
+     * and combine it into a [MessagesState] that holds all the information required for the screen.
      */
-    private fun observeChannel() {
+    private fun observeMessages() {
```

Comment:
Maybe we could call it `observeChannelState`.
It seems more technically correct.